src/goto-programs/lazy_goto_model.cpp

```diff
@@ -207,6 +208,24 @@ void lazy_goto_modelt::load_all_functions() const
 
 bool lazy_goto_modelt::freeze()
 {
+  messaget msg(language_files.get_message_handler());
+  journalling_symbol_tablet symbol_table=
+    journalling_symbol_tablet::wrap(this->symbol_table);
+  if(language_files.final(symbol_table))
+  {
+    msg.error() << "FINAL STAGE CONVERSION ERROR" << messaget::eom;
+    return false;
```

Comment:
This is not the usual convention in this code base: return `true` in case of errors.

Reply:
The reality is that both are used quite a lot in the code base. This is not part of the API of this class so I'll leave it unless you really care.

Reply:
Any examples of cases where `false` is used to signify an error? (Yes, I do really care.)